ui/webui/src/components/users/Accounts.scss

```diff
@@ -0,0 +1,4 @@
+// Span disk encryption password fields to take slightly more width than the default
```

Comment:
Wrong comment.

Reply:
[webui: fix password strength indicator layout in horizontal form](https://github.com/rhinstaller/anaconda/pull/5280/commits/1588dad011eebe7cfdf4d65d6d4a6a697914bd23)